backend/routes/admin_routes.py

```
@@ -59,6 +59,30 @@ def verify_admin_access():
 
+
+def verify_superadmin_access():
+    """
+    Verify JWT authentication and superadmin role access.
+
+    Returns:
+        tuple: (error_response, error_code) if verification fails, (None, None) if successful
+    """
+    try:
+        verify_jwt_in_request()
+    except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)